src/server/routes/verifyEmail.ts

```diff
@@ -146,7 +144,6 @@ router.post(
 
 router.get(
   '/verify-email/:token',
-
```

Comment:
Do we know what was up with this formatting before? Some kind of mass find + replace operation maybe?

I see that `yarn test` doesn't run prettier. Was that why this wasn't caught?

Reply:
I think it was from when I moved from having the `rateLimitMiddleware` included at this level to moving it up to the router so we didn't have to add it to every single route. Odd that the linter didn' catch it

Reply:
I think only ESLint runs on the CI, not Prettier.

Reply:
👍